content/blog/2018/01/2018-01-15-overhaul-of-manage-jenkins-page.adoc

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: "Overhaul of Manage Jenkins page"
+tags:
+- jenkins
+- ui
+- restyling
+author: recena
+---
+
+== Overview
+
+Recently we have introduced some UI improvements around to the Manage Jenkins page. The visual changes are very subtle but behind of them, there are interesting benefits.
+
+Some of the goals that we have tried to achieve:
+
+* Apply a https://en.wikipedia.org/wiki/Semantic_HTML[semantic HTML]
+* Remove the `<table>` tag usage for implementing layouts and content structures. Read this https://www.hotdesign.com/seybold[article] if you want to know reasons and / or arguments.
+* Small re-style focused on spacing, margins, composition, etc..
+* Accesibility
+
+In order to provide a quick view of the visual changes, let's take a look at these screenshots.
+
+=== System tray with administrative messages (before)
+
+image:/images/post-images/2018-01-15-JENKINS-43786/JENKINS-43786_3-before.png[role="center"]
+
+=== System tray with administrative messages (after)
+
+image:/images/post-images/2018-01-15-JENKINS-43786/JENKINS-43786_2.png[role="center"]
+
+=== Manage Jenkins page (before)
+
+image:/images/post-images/2018-01-15-JENKINS-43786/JENKINS-43786_4-before.png[role="center"]
+
+=== Manage Jenkins page (after)
+
+image:/images/post-images/2018-01-15-JENKINS-43786/JENKINS-43786_1.png[role="center"]
+
+In the following section you will find information about how this change can affect to the current implementations of the https://jenkins.io/doc/developer/extensions/jenkins-core/#administrativemonitor[Administrative Monitors].
```

Comment:
The passive voice sounds better IMHO ==> "Information about how this change can affect to the current implementations can be found in the following section"